### PR TITLE
Refactor calculate Timeshift Window For SLA

### PR DESCRIPTION
**Pull request by hehnle**

### CODE REVIEW

File: carbon-reductor-core/src/main/java/de/envite/greenbpm/carbonreductor/core/domain/service/DelayCalculatorService.java [hunk 1]
```diff
@@ -67,14 +67,13 @@ public CarbonReduction calculateDelay(CarbonReductorConfiguration input) throws
     }
 
     Timeshift calculateTimeshiftWindowForSLA(CarbonReductorConfiguration input) {
-        // maximumProcessDuration - milestone - remainingDuration - now
+        // lastPossibletEndDateTime - remainingDuration - now
         // (milestone + maximumDuration) - milestone - remaining - (now - milestone)
-        OffsetDateTime maximumDurationDateTime = input.getMilestone().asDate().plus(input.getMaximumProcessTimeshift().getValue().toMillis(), ChronoUnit.MILLIS);
-        long msSinceMilestone = OffsetDateTime.now(ZoneOffset.UTC).toInstant().toEpochMilli() - input.getMilestone().asDate().toInstant().toEpochMilli();
+		OffsetDateTime lastPossibleEndDateTime = input.getMilestone().asDate().plus(input.getMaximumProcessTimeshift().getValue().toMillis(), ChronoUnit.MILLIS);
```
Comment (lwluc) [inline carbon-reductor-core/src/main/java/de/envite/greenbpm/carbonreductor/core/domain/service/DelayCalculatorService.java:72]: Formatting still has some potential here

**Review by lwluc — CHANGES_REQUESTED** [review]

File: carbon-reductor-core/src/main/java/de/envite/greenbpm/carbonreductor/core/domain/service/DelayCalculatorService.java (hunk 1 above)
Comment (hehnle) [reply carbon-reductor-core/src/main/java/de/envite/greenbpm/carbonreductor/core/domain/service/DelayCalculatorService.java:72]: That is?

Comment (hehnle) [reply carbon-reductor-core/src/main/java/de/envite/greenbpm/carbonreductor/core/domain/service/DelayCalculatorService.java:72]: I don't know why github shows an indention. My IDE doesn't

File: carbon-reductor-core/src/main/java/de/envite/greenbpm/carbonreductor/core/domain/service/DelayCalculatorService.java [hunk 2]
```diff
@@ -67,14 +67,13 @@ public CarbonReduction calculateDelay(CarbonReductorConfiguration input) throws
     }
 
     Timeshift calculateTimeshiftWindowForSLA(CarbonReductorConfiguration input) {
-        // maximumProcessDuration - milestone - remainingDuration - now
+        // lastPossibletEndDateTime - remainingDuration - now
         // (milestone + maximumDuration) - milestone - remaining - (now - milestone)
```
[the remark this answers is not in the document]
Comment (hehnle) [reply carbon-reductor-core/src/main/java/de/envite/greenbpm/carbonreductor/core/domain/service/DelayCalculatorService.java]: done

**Review by lwluc — CHANGES_REQUESTED** [review]
Squash Commits?

**Review by lwluc — APPROVED** [review]

**Review by maxbehr801 — APPROVED** [review]

**Pull request merged**